README.md

```diff
@@ -425,3 +431,381 @@ Now that it's done, if we go back to the repository the python script will be vi
 ![github.com main branch after merge](./images/web_main_merged.png)
 
 Now people not contributing but using your code can simply download from the main branch and are sure the code will work.
+
+## Git on the command line ##
+
+So far we have seen how to use git using GUI tools as well as navigate around the GitHub interface. Next, we will take a look at how we can use git on the terminal.
+
+To execute Git commands in your computer, you must open a command shell (also known as command prompt, terminal, and command line) of your preference. Here are some suggestions:
+
+- **For macOS users:**
+  - Built-in: Terminal. Press ⌘ command + space and type “terminal” to find it.
+  - iTerm2, which you can integrate with zsh and oh my zsh for color highlighting, among other handy features for Git users.
+- **For Windows users:**
+  - Built-in: cmd. Click the search icon on the bottom navigation bar on Windows and type cmd to find it.
+  - PowerShell: a Windows “powered up” shell, from which you can execute a greater number of commands.
+  - Git Bash: it comes built into Git for Windows.
+- **For Linux users:**
+  - Built-in: Linux Terminal.
+
+### Installing and configuring git on the terminal ### 
+
+There are a lot of resources online which will guide you how to download, insntall and configure git on your local machine. A quick Google search ‘How to install git on [your preferred OS]” will reveal a number of resources. Here is a useful link: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
```

Comment:
The start and end quotes around **How to install git on [your preferred OS]** are different.